platform/ui/src/components/userPreferencesForm/HotKeysPreferences.styl

```diff
@@ -1,7 +1,5 @@
 .HotKeysPreferences
   display: flex;
-  margin-right: -15px;
-  margin-left: -15px;
```

Comment:
The negative margin trick is how the bootstrap "row" class works. Bye bye negative margin 👋